### PR TITLE
doc: Add ADOPTERS.md

### PR DESCRIPTION
**Pull request by wenyongh**

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+| [Moonbit](https://www.moonbitlang.com) | [@peter-jerry-ye](https://github.com/peter-jerry-ye) | ![Undisclosed](https://img.shields.io/badge/-Undisclosed-blue?style=flat) | MoonBit is an end-to-end programming language toolchain for cloud and edge computing using WebAssembly. |
+| [Microsoft](https://www.microsoft.com) | [@Mossaka](https://github.com/Mossaka)                       | ![production](https://img.shields.io/badge/-production-blue?style=flat)                                                       | Hyperlight runs Wasm workloads in VMs without OS and kernel, it is a solution for improving the management and security of Wasm workloads on Azure. |
+| [Midokura](https://www.midokura.com)                          |   [@yamt](https://github.com/yamt)                                     | ![Undisclosed](https://img.shields.io/badge/-Undisclosed-blue?style=flat) | WEdge: the next-generation Edge AI sensing platform                                                   |
```
Comment (yamt) [inline ADOPTERS.md:16]: it seems that the name "WEdge" is no longer used.
i'm not sure what's the latest name.
please just remove "WEdge: " for now.

**Comment by yamt** [comment]
are these badges provided by BA?
it's nicer if different badges have different colors.

File: ADOPTERS.md (hunk 1 above)
Comment (wenyongh) [reply ADOPTERS.md]: Done.

**Comment by wenyongh** [comment]
> are these badges provided by BA? it's nicer if different badges have different colors.

It is auto generated by https://img.shields.io with putting the text, color, style in the link, e.g. https://img.shields.io/badge/-production-blue?style=flat, or https://img.shields.io/badge/-production-orange?style=flat

> are these badges provided by BA? it's nicer if different badges have different colors.

Done, changed the color of Undisclosed to orange.

File: ADOPTERS.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+| Organization                           | Contact                                                      | Status                                                       | Description of Use                                           |
+| -------------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| [Alibaba](https://www.alibaba.com)     | [@johnlanni](https://github.com/johnlanni)                   | ![production](https://img.shields.io/badge/-production-blue?style=flat) | Higress is a next-generation cloud-native gateway built on the core of open-source Istio + Envoy based on Alibaba's internal Envoy Gateway practice. |
+| [Amazon](https://www.amazon.com)       | [@loganek](https://github.com/loganek)                       | ![production](https://img.shields.io/badge/-production-blue?style=flat) | Amazon Prime Video is an American subscription video on-demand over-the-top streaming and rental service of Amazon offered as a standalone service or as part of Amazon's Prime subscription. |
```
Comment (loganek) [inline ADOPTERS.md:9]: I'd remove `American` from the sentence; the service is available globally. Also, Prime Video does not only offer video on demand but also live content.

I think something like that would do:
> Prime Video is a global streaming service by Amazon that provides on-demand access to a vast library of movies, TV shows, and original programming, as well as live content.

**Review by yamt — APPROVED** [review]
lgtm

**Pull request merged**